Automattic/wp-calypso
# WebPreview: add `is currently showing` to redux

In preparation of a `GuidedTours` architecture rewrite; we need this information to be available in redux.

Props @ehg, @mcsf 

To test, 

- open a site preview, post preview, page preview, theme preview
- use the [Redux DevTools](https://chrome.google.com/webstore/detail/redux-devtools/lmhkpmbekcpmknklioeibfkpmmfibljd) to check that `PREVIEW_IS_SHOWING` is set to `true` when showing the preview and to `false` when closing it again

Test live: https://calypso.live/?branch=add/webpreview-showing
